src/API/API.test.js

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { waitFor, mockFn } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import API from "./API";
+import MutationObserver from "@sheerun/mutationobserver-shim";
+window.MutationObserver = MutationObserver;
+
+
+describe('API methods', () => {
+  describe('Helper Functions', () => {
+
+    it(`should be able to determine which key to use on
+    a fetch response based on the end of the URL`, () => {
+      const movies = API.findRelevantData('movies')
+      const singleMovie = API.findRelevantData('movies/:movie_id')
+      const videos = API.findRelevantData('movies/:movie_id/videos')
+      const ratings = API.findRelevantData("users/:user_id/ratings")
+
+      expect(movies).toBe('movies')
+      expect(singleMovie).toBe('movie')
+      expect(videos).toBe('videos')
+      expect(ratings).toBe('ratings')
+    })
+
+    // it(`should return an error if it gets a bad path`, () => {
+    //   const getItWrong = () => {
```

Comment:
If you're defining `getItWrong` as a function, you have to give it a parameter to receive its [argument](https://github.com/GreysonElkins/roasted-tomahtoes/pull/47/files#diff-c17c48e9bbd2ad4df4bc878b9fc213b7R29). I'm not sure if that was part of the issue? It'd be helpful to see some error messages for this test.